### PR TITLE
Validate README instructions

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,41 @@
-# odoh-client
+# odoh-client-go
```
Comment (cjpatton) [inline README.md:1]: Somewhat opinionated: my feeling is that the README title should match the package name. I'll revert if you disagree.

File: README.md [hunk 2]
```diff
@@ -1,35 +1,41 @@
...
 
-`odoh-client` is a command line interface as a client for performing oblivious dns-over-https queries, based on [chris-wood/odoh-client](https://github.com/chris-wood/odoh-client).
+`odoh-client` is a command line interface as a client for making [Oblivious DNS-over-HTTPS](https://tools.ietf.org/html/draft-pauly-dprive-oblivious-doh-03) queries. The implementation is based on [chris-wood/odoh-client](https://github.com/chris-wood/odoh-client).
 
 It currently supports the following functionalities:
 
-- [x] DoH Query : `odoh-client doh --domain www.cloudflare.com. --dnsType AAAA`
-- [x] ODoH Query: `odoh-client odoh --domain www.cloudflare.com. --dnsType AAAA --target <target>`
-- [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnsType AAAA --target <target> --use-proxy true --proxy <proxy>`
+- [x] DoH Query: `odoh-client doh --domain www.cloudflare.com. --dnstype AAAA --target <target>` where `<target>` is the XXX, e.g., `odoh.cloudflare-dns.com`.
```
Comment (cjpatton) [inline README.md]: Replace `XXX` with a description of the target.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,41 @@
...
+- [x] ODoH Query: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target>`
+- [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target> --proxy <proxy>` where `<proxy>` is the XXX, e.g., `odoh1.surfdomeinen.nl`.
```
Comment (cjpatton) [inline README.md]: Replace XXX with a description of the proxy.

File: README.md (hunk 2 above)
Comment (chris-wood) [inline README.md]:
```suggestion
- [x] DoH Query: `odoh-client doh --domain www.cloudflare.com. --dnstype AAAA --target <target>` where `<target>` is the name of the target resolver, e.g., `odoh.cloudflare-dns.com`.
```

File: README.md (hunk 3 above)
Comment (chris-wood) [inline README.md]:
```suggestion
- [x] ODoH Query via Proxy: `odoh-client odoh --domain www.cloudflare.com. --dnstype AAAA --target <target> --proxy <proxy>` where `<proxy>` is the name of a proxy server.
```

**Review by chris-wood — APPROVED** [review]

**Pull request merged**